frontend/.eslintrc.cjs

```diff
@@ -1,21 +1,28 @@
-{
-  "env": {
-    "browser": true,
-    "es2021": true
+module.exports = {
```

Comment:
Just for my edification, can you explain what this change is doing?

Reply:
Great questions!

I converted the `.eslintrc` config file from JSON to `.cjs` in order to be able to access `__dirname`, which is a global variable in JS that stores the current directory name. This was to make path aliases possible. It's a `.cjs`, using `module.exports`, instead of a `.js` file because it didn't work as a `.js` file lol

Reply:
Gotcha! I saw the option to make eslint a js file but defaulted to JSON. That's really interesting.